packages/components/src/HttpError/HttpError.component.js

```diff
@@ -21,6 +23,9 @@ function HttpError(props) {
 			>
 				<h1>{title}</h1>
 				<p>{message}</p>
+				{backAction && (
+					<Action icon="talend-arrow-left" label={backLabel} link onClick={backAction} />
```

Comment:
I think you should align the api with action api, instead of taking onclick + label.
2 choices : 
* you allow to put any action, you just take a props.action and spread it to `<Action {...props.action} />`
or
* you only allow a back action, you take a props.backAction and just enforce the link style + arrow icon `<Action {...props.backAction} icon="talend-arrow-left" link />`

Reply:
Good point! I chose first way